spec/project-spec.coffee

```diff
@@ -197,6 +199,43 @@ describe "Project", ->
           atom.project.bufferForPath("b").then (anotherBuffer) ->
             expect(anotherBuffer).not.toBe buffer
 
+  describe ".repositoryForDirectory(directory)", ->
+    # it() causes beforeEach() in spec-helper.coffee to be run, which reassigns
+    # atom.project. Therefore, the atom.project used in waitsForPromise() will
+    # be different than the atom.project used in a subsequent it(). To work
+    # around this issue, we use runs() instead of it().
+
+    it "there is no repository for /tmp even though there are RepositoryProviders", ->
```

Comment:
We try to phrase these so that they read nicely starting with an `it`. For this one, I'd say something like `it "resolves to null for the /tmp directory"`.